pkg/kudoctl/cmd/init.go

```diff
@@ -153,7 +166,7 @@ func (initCmd *initCmd) run() error {
 			initCmd.client = client
 		}
 
-		if err := cmdInit.Install(initCmd.client, opts); err != nil {
+		if err := cmdInit.Install(initCmd.client, opts, initCmd.crdOnly); err != nil {
 			if apierrors.IsAlreadyExists(err) {
 				clog.Printf("Warning: KUDO is already installed in the cluster.\n" +
 					"(Use --client-only to suppress this message)")
```

Comment:
I am just wondering what should the behavior of `kudo init --crd-only --wait` be (it's few lines under this one, I cannot put a comment there). I would probably either:
- make it an invalid option
- make it so that it finished immediately

Slightly preferring first option

Reply:
makes sense